README.md

```diff
@@ -159,9 +159,9 @@ To run tests, run the following command:
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/215242460-e92f4c33-c4bc-493b-a113-fd4d09f9a111.png)

- [ ] Instead of **Author 1** Kindly put your name to make your readme professional :+1: